core/server/services/auth/index.js

```diff
@@ -1,13 +1,20 @@
 var passport = require('./passport'),
+    auth = require('./auth'),
     authorize = require('./authorize'),
     authenticate = require('./authenticate'),
+    session = require('./session'),
     oauth = require('./oauth');
 
+/* TODO: Remove when v0.1 is dropped */
```

Comment:
If you would use

```
/**
  * @TODO
  *
*/
```

IDE's is able to highlight.

Suggestion: You could use group this file a little bit to avoid having twice the same todo 🙂

Reply:
Sure thing :small_airplane: